pkg/cmd/workspace/ssh.go

```diff
@@ -78,7 +79,36 @@ var SshCmd = &cobra.Command{
 		}
 
 		if !workspace_util.IsProjectRunning(workspace, projectName) {
-			views.RenderInfoMessage(fmt.Sprintf("Project '%s' from workspace '%s' is not in running state", projectName, workspace.Name))
+			var yesFlag bool
```

Comment:
Please add a flag to the command that will allow the user to skip this prompt (e.g. `--auto-start`)

Reply:
done